spec/models/team_spec.rb

```diff
@@ -147,6 +147,86 @@
     end
   end
 
+  describe 'limit of coaches' do
+    let(:team) { create :team }
+    let(:new_user) { create :user }
+    let(:new_coach) { -> {{ name: 'coach', team_id: team.id, user_id: new_user.id }} }
```

Comment:
Personal taste: I tend to make "generator spec helpers" real methods with `def foo(arg1)` instead of defining a proc/lambda in a `let`.

Reply:
Nice! In your opinion, where would these be defined? The same spec file?